server/src/common/dto/user.dto.ts

```diff
@@ -0,0 +1,4 @@
+export interface UserDto {
```

Comment:
UserAccessDTO -> not an interface but a class. Define the class with decorators. 

Eg:
`IsBoolean()
IsOptional()`